portfolio/target/portfolio-1/calendar.js

```diff
@@ -1,86 +1,141 @@
-/** 
+/**
  * Models a calendar event that is displayed on the UI.
- * This class is useful when the event information needs to be sent 
- * to the calendar servlet through a POST request. 
+ * This class is useful when the event information needs to be sent
+ * to the calendar servlet through a POST request.
  * This class contains all the event information that can be easily
- * converted into a JSON string. 
- * All fields are required. 
- * @param name: of type String, name of event
- *        startTime: of type Date in format "Day Month Date Year HH:MM:SS GMT-Time Zone" 
- *        endTime: of type Date in format "Day Month Date Year HH:MM:SS GMT-Time Zone"
+ * converted into a JSON string.
  */
 class CalendarEvent {
+  /**
+   * All fields are required.
+   * @param name: of type String, name of event
+   * @param startTime: of type Date in format "Day Month Date Year HH:MM:SS
+   *    GMT-Time Zone" 
+   * @param endTime: of type Date in format "Day Month Date Year HH:MM:SS
+   *    GMT-Time Zone"
+   */
   constructor(name, startTime, endTime) {
     this.name = name;
     this.startTime = startTime;
     this.endTime = endTime;
   }
 }
 
-/** 
+let allEvents = new Set(); 
```

Comment:
Can this be `const` instead of `let`?

Reply:
Updated.

Reply:
I still see `let`. Do you have another change to push?